app/views/pages/privacy_policy.html.erb

```diff
@@ -0,0 +1,193 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+  <h1 class='govuk-heading-xl'>
+    Privacy Notice: Teachers: claim back your student loan repayments
+  </h1>
+
+    <h2 class='govuk-heading-l'>
+      Who we are
+    </h2>
+
+    <p class="govuk-body">
+      This work is being carried out by the Department for Education (DfE) Digital, which is a part of DfE. DfE engages the
+      private company dxw to help improve and provide the service. For the purpose of data protection legislation, DfE
```

Comment:
Also Paper (and Tekin).

Reply:
Added in Paper I think Tekin is ok as sub contracted through us.